docs/world api.md

```diff
@@ -446,8 +446,8 @@ In addition, the following methods can be implemented and attributes can be set
 
 ```python
 def generate_early(self) -> None:
-    # read player settings to world instance
-    self.final_boss_hp = self.world.final_boss_hp[self.player].value
+  # read player settings to world instance
+  self.final_boss_hp = self.multiworld.final_boss_hp[self.player].value
```

Comment:
We're trying to follow pep8 aren't we? Wouldn't we also want to follow it in the documentation?
Why the change to a 2-space indentation?

Reply:
uhhh this definitely wasn't on purpose as i literally just ctrl+f for world...